alerts/ui/src/components/filters/FilterPills.jsx

```diff
@@ -22,6 +42,21 @@ const FilterPills = () => {
             closeable
             onClose={() => removeActiveFilter(key, value)}
             key={`${key}:${value}`}
+            onClick={() => pauseFilter(key, value)}
+          />
+        ))
+      })}
+      {Object.entries(pausedFilter).map(([key, values]) => {
```

Comment:
If I read this correctly you're moving all paused filters to the end of the Pill Stack. That can lead to undesirable UX because the Pill someone clicked to pause it gets yanked away from its original position. It's much better to leave it in place and only change its state.

Reply:
done